src/app/cluster/details/shared/version-change-dialog/template.html

```diff
@@ -55,6 +55,16 @@
       Upgrade Machine Deployments
     </mat-checkbox>
   </div>
+  <div *ngIf="!isDynamicKubletConfigSupportedInUpgrade && this.nodesSupportDynamicKubeletConfig"
```

Comment:
avoid using `this` in template.

`public/protected BOTH`

- Any getter/setter property or method can be directly used in template without `this` as template is automatically binded to that class instance